wrappers/javascript/yarn.lock

```diff
@@ -1456,7 +1290,15 @@
   dependencies:
     "@types/ref-napi" "*"
 
-"@types/ref-struct-di@*", "@types/ref-struct-di@^1.1.6":
+"@types/ref-napi@*":
+  name "@types/2060.io__ref-napi"
```

Comment:
I think this replacement came from the previous types config that might be cached somewhere (@types/2060.io__ref-napi does not actually exist). I could solve this locally by deleting it from yarn.lock (or removing the yarn lock file entirely) and running `yarn install` again. That way it will retrieve the actual `@types/ref-napi` package as it should be in the first place.